pg_analyse/toolbox.py

```diff
@@ -1,6 +1,6 @@
 from typing import List, Union, Set, Dict
 
-import psycopg2
+import psycopg
```

Comment:
Let's make it in a backward compatible manner for those who has no chance to upgrade

```
try:
    import psycopg

except ImportError:
    import psycopg2 as psycopg

```

Here and in setup.py it seems.

Reply:
Updated my PR according to your comment, please check